Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml

```
@@ -90,6 +97,11 @@
     tags:
     - HandGrenade
     - GrenadeFlashBang
+    - HandGrenadeAmmo #Sunrise-Start
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)